Update runtime to 20.08

Eonfge opened the pull request:
Fixes #4

tanuk commented:
The commit message should explain why the intltool module was added and why adwaita-icon-theme was updated. If the runtime update doesn't require those changes, the changes should be in separate commits.

Eonfge commented:
As the Maintainer said to me, and I'll say to you:

> PR welcome!

I've shown how to get this package up to date using the latest runtime, while the project has not been updated for a year. You can take it from there.

wjt commented:
@Eonfge for the record @tanuk is the upstream maintainer of this app. He is reviewing your contribution.

(I made that rather flippant comment on #4 because I don't find "Update to $RUNTIME" issues helpful. The Flatpak was last updated in June 2019 because that was when the last upstream release was made. While 18.08 was fully supported there was no reason to update the runtime. Now that 18.08 is EOLed it is indeed more interesting to update to 20.08.)

ford-prefect commented:
FWIW, I'm happy to split the commit etc. if there's an explanation of the rationale for the `intltool` thing (and if not, will dig in and figure this out -- the work you've done is still welcome and appreciated @Eonfge

Eonfge committed:
appdata: update runtime

We must include the intltool from the shared modules because the latest runtime requires a version, newer than provided by the base project.

ford-prefect commented:
bot, build

Eonfge commented:
Think we started a bit on a wrong foot here: the 'flippant comment' did prompt me to waste half an hour of my live to show off, so I didn't assume any good faith. My bad. This is not my project so I should make a reasonable effort to follow your procedures. I'll try to do so in the future.

I've updated the pull request with some more contextual information. Updating the logo and gtkmm is not part of this feature, so I guess that will warrant a different pull request?

ford-prefect commented:
Thanks @Eonfge! Another PR with separate commits for those two changes would be great.

One comment: this is not a big deal, but usually you would use the `appdata: ` prefix in the commit message if you're only changing appdata, etc. I've merged this anyway since that's a nitpick.

wjt commented:
My bad, more: thank you for persevering in face of some quite discouraging comments from me, and please forgive me for those comments.

Eonfge commented:
We all have our grumpy moments and I was no better. All forgiven and lets keep helping each other in the future
